examples/jlang-parser/src/j.pest

```diff
@@ -0,0 +1,25 @@
+program = _{ SOI ~ "\n"* ~ (stmt ~ "\n"+) * ~ stmt? ~ EOI }
+
+stmt = _{ expr }
+
+expr = { assgmtExpr | monadicExpr | dyadicExpr | string | terms }
+monadicExpr = { action ~ expr }
+dyadicExpr = { (monadicExpr | terms) ~ action ~ expr }
+
+assgmtExpr = { ident ~ "=:" ~ expr }
+terms = { term+ }
+term = _{ decimal | integer | ident | "(" ~ expr ~ ")" }
```

Comment:
To keep style consistency within the book, it would be nice to use a more vertically sprawling format for the pest files like in the [JSON grammar](https://github.com/pest-parser/book/blob/master/examples/json-parser/src/json.pest). This means that some of these lines could be split in a similar manner. (same styling should also be applied in markdown)